cmd/root.go

```diff
@@ -86,7 +86,8 @@ func init() {
 
 	// Options for optional ephemeral system properties.
 	f.BoolVarP(&scanner.Swap, "swap", "s", false, "capture swap entries")
-	f.BoolVarP(&scanner.Ephemeral, "ephemeral", "e", false, "capture all ephemeral properties e.g. swap, filesystems and so on")
+	f.BoolVarP(&scanner.Mounts, "mounts", "m", false, "capture filesystem mounts")
+	f.BoolVarP(&scanner.Ephemeral, "ephemeral", "e", false, "capture all ephemeral properties e.g. swap, mounts and so on")
```

Comment:
`ephemeral` seems to contains only SwapEntries, no "mounts and so on"? Or am I misreading?